web/modules/custom/usa_twig_vars/usa_twig_vars.module

```diff
@@ -300,3 +308,44 @@ function usa_twig_vars_after_build($form, &$form_state) {
   }
   return $form;
 }
+
+/** Helper function to check if node should be excluded from contact center based on path. Updates the exclude from url checkbox on edit pages.  */
+
+function usa_twig_vars_check_exclude_from_contact_center($nodeToCheck) {
+  $pathAliasToCheck = $nodeToCheck->toUrl()->toString();
+  if (str_contains($pathAliasToCheck, '/es/articulos/') || str_contains($pathAliasToCheck, '/features/')) {
+    $old_checkbox = $nodeToCheck->get('field_exclude_from_contact_cente')[0]->value;
+
+    $nodeToCheck->set('field_exclude_from_contact_cente', 1);
+    $nodeToCheck->save();
+
+    $new_checkbox = $nodeToCheck->get('field_exclude_from_contact_cente')[0]->value;
+  }
+}
+
+/**
+ * Implements hook_views_post_execute()
+ * When a user goes to this view it will call usa_twig_vars_check_exclude_from_contact_center() on all the nodes to check the box for all features to the exclude from contact center
+ */
+
+function usa_twig_vars_views_post_execute($view) {
+  if ($view->id() == 'features' && ($view->current_display == 'all_paths')) {
+    $new_result = $view->result;
+    $num_results = count($new_result);
+
+    for ($res = 0; $res < $num_results; $res++) {
+      $nid = $new_result[$res]->_entity->nid[0]->value;
+      $node = Node::load($nid);
+      usa_twig_vars_check_exclude_from_contact_center($node);
+    }
+  }
+}
+
+function usa_twig_vars_path_alias_update(PathAliasInterface $path) {
```

Comment:
This doesn't catch brand-new pages, and at first I thought that was the thing everyone was having trouble getting at. But, armed with the `_update` hook you identified here, I decided to see whether adding a `usa_twig_vars_path_alias_insert` function would do the trick. It seems to work!!!  I just copied this function and renamed it to try it out. 

 So, could we add an `_insert` function, please?